### ISSUE TITLE
gribscan in levante

### ISSUE THREAD

**Issue by endutra** [nextGEMS/nextGems_Cycle2#5]
Hi @xabipedru  

Is there a special module to have gribscan ? 
when doing module load python3 is not available. I just installed it locally with pip install --user gribscan

**Comment by xabipedru** [comment]
Yes that's right, you should install it first with pip. If in the command line at levante: 
`python -m pip install gribcan`

From a jupyther notebook in levante:
`! pip install gribscan`
